starlark/interp.go

```diff
@@ -52,6 +52,12 @@ func (fn *Function) CallInternal(thread *Thread, args Tuple, kwargs []Tuple) (Va
 		defer fmt.Println("Leaving ", f.Name)
 	}
 
+	// Spill indicated locals to cells.
+	// Each cell is a separate alloc to avoid spurious liveness.
```

Comment:
Thanks for keeping these separate. I've seen a couple bad memory leaks in V8 that were the consequence of allocating an array for captured variables instead of individual cells.